<launch>

  <node name="line_detection_fu_node" pkg="line_detection_fu" type="line_detection_fu_node" > 
    <param name="cam_w"                         type="int"    value="1280"/>  
    <param name="cam_h"                         type="int"    value="720"/> 	
	<param name="proj_y_start"            	    type="int" 		value="50"/>
  	<param name="proj_image_h"            	    type="int" 		value="80"/>
  	<param name="proj_image_w"            	    type="int" 		value="160"/>
  	<param name="proj_image_horizontal_offset"  type="int"    value="0"/>
    <param name="roi_top_w"            	        type="int" 		value="160"/>
  	<param name="roi_bottom_w"            	    type="int" 		value="86"/>
    <param name="minYRoi"                       type="int"    value="0"/>
    <param name="maxYDefaultRoi"                type="int"    value="79"/>
    <param name="maxYPolyRoi"                   type="int"    value="79"/>
    <param name="defaultXLeft"                  type="int"    value="8"/>
    <param name="defaultXCenter"                type="int"    value="56"/>
    <param name="defaultXRight"                 type="int"    value="104"/>
    <param name="interestDistancePoly"          type="int"    value="30"/>
    <param name="interestDistanceDefault"       type="int"    value="26"/>
    <param name="iterationsRansac"              type="int"    value="10"/>
    <param name="proportionThreshould"          type="double" value="0.5"/>
    <param name="m_gradientThreshold"           type="int"    value="10"/>
    <param name="m_nonMaxWidth"                 type="int"    value="10"/>
    <param name="laneMarkingSquaredThreshold"   type="int"    value="36"/>
    <param name="angleAdjacentLeg"              type="int"    value="75"/>
    <param name="scanlinesVerticalDistance"     type="int"    value="1"/>
    <param name="scanlinesMaxCount"             type="int"    value="100"/>
    <param name="polyY1"                        type="int"    value="75"/>
    <param name="polyY2"                        type="int"    value="60"/>    
    <param name="polyY3"                        type="int"    value="40"/>    
    <param name="detectLaneStartX"              type="int"    value="75"/>
    <param name="maxAngleDiff"                  type="int"    value="5"/>
  </node>
</launch>
